test/message/test_runner_output_cli.out

```diff
@@ -0,0 +1,652 @@
+TAP version 14
```

Comment:
This output looks incorrect. It looks like it could be the same as `test/message/test_runner_output.out`. I'm not sure that the test is working properly.

Reply:
Fixed.